ginga/tests/test_colordist.py

```diff
@@ -0,0 +1,63 @@
+"""Test ColorDist.py"""
+from __future__ import division
+
+import numpy as np
+import pytest
+
+from ginga import ColorDist as cd
+
+
+# Some of the code is based on https://github.com/ejeschke/ginga/pull/346
```

Comment:
Do pull requests live on forever, even after they've merged or deleted?  Not sure this comment URL will be valid in the future.

Reply:
Yes, merged PR still exists in your repo until GitHub implodes (touch wood). In fact, deleted branches are not really deleted, they are just very hard to access unless you remember the commit hash and/or branch name.